cmd/containers-storage/create.go

```diff
@@ -22,14 +25,110 @@ var (
 	paramMetadata     = ""
 	paramMetadataFile = ""
 	paramCreateRO     = false
+	paramHostUIDMap   = false
+	paramHostGIDMap   = false
+	paramUIDMap       = ""
+	paramGIDMap       = ""
+	paramSubUIDMap    = ""
+	paramSubGIDMap    = ""
 )
 
+func paramIDMapping() (*storage.IDMappingOptions, error) {
+	options := storage.IDMappingOptions{
+		HostUIDMapping: paramHostUIDMap,
+		HostGIDMapping: paramHostGIDMap,
+	}
+	if paramHostUIDMap && (len(paramUIDMap) > 0 || paramSubUIDMap != "") {
+		return nil, fmt.Errorf("host uid map specified along with UID mapping")
+	}
+	if paramHostGIDMap && (len(paramGIDMap) > 0 || paramSubGIDMap != "") {
+		return nil, fmt.Errorf("host gid map specified along with GID mapping")
+	}
+	if paramSubUIDMap != "" && paramSubGIDMap == "" {
+		return nil, fmt.Errorf("subuid map specified but no subgid map")
```

Comment:
Changed this so that if you specify one but not the other, we assume the same name for both.  It slightly reduces the amount of work configuring this requires, and `useradd` already adds entries to both files.